### PR TITLE
Inherits schema to inherit elements

### PR DESCRIPTION
**Pull request by bjormel**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: en/schemas.html [hunk 1]
```diff
@@ -270,11 +270,9 @@ <h2 id="multiple-schemas">Multiple schemas</h2>
 </p>
 <pre>
 $ENDPOINT/search/?
-  yql=select * from sources * where title contains "bob"&amp;
+  vespa query "select * from sources * where title contains \"bob\""&amp;
```
Comment (kkraune) [inline en/schemas.html]:
```suggestion
  vespa query "select * from sources * where title contains \"bob\""
```

File: en/schemas.html [hunk 2]
```diff
@@ -195,7 +195,7 @@ <h3 id="fieldset">fieldset</h3>
 }
 
 $ENDPOINT/search/?
```
Comment (kkraune) [inline en/schemas.html]: remove the $ENDPOINT/search/? as well?

**Review by kkraune — APPROVED** [review]
Great! I have not tested schema inheritance myself, I assume this works well

**Comment by bjormel** [comment]
hmm, since we have the this new AI powered doc search, I guess that we should use complete examples. Will add doc defs to schema examples where they are not defined

**Pull request merged**